common/tests/threadpool_int/threadpool_int.c

```diff
@@ -843,7 +845,7 @@ TEST_FUNCTION(timer_cancel_restart_works_runs_periodically)
     ASSERT_IS_NOT_NULL(timer);
 
     // Timer should run 4 times in about 2.1 seconds
```

Comment:
>// Timer should run 4 times in about 2.1 seconds [](http://example.com/codeflow?start=4&length=48)

These comments should be fixed.  Here and on line 859

Reply:
done